packages/ui-react/src/components/HeroShelf/HeroShelf.tsx

```diff
@@ -27,20 +27,28 @@ const HeroShelf = ({ playlist, loading = false, error = null }: Props) => {
   const { t } = useTranslation('common');
   const breakpoint = useBreakpoint();
   const isMobile = breakpoint <= Breakpoint.sm;
-  const scrolledDown = useScrolledDown(500, 200);
+  const posterRef = useRef<HTMLDivElement>(null);
   const [direction, setDirection] = useState<'left' | 'right' | null>(null);
   const [animationPhase, setAnimationPhase] = useState<'init' | 'start' | 'end' | null>(null);
+  const [isSwipeAnimation, setIsSwipeAnimation] = useState(false);
 
-  const slideTo = (toIndex: number) => {
+  useScrolledDown(50, isMobile ? 200 : 700, (progress: number) => {
+    if (posterRef.current) posterRef.current.style.opacity = `${Math.max(1 - progress, isMobile ? 0 : 0.1)}`;
```

Comment:
Should we also fade away the pagination?

Reply:
I didn't find it necessary anymore. Before they were also fixed, but they scroll away now.